itunes-connect.csv

```diff
@@ -0,0 +1,27 @@
+Name,Aware
+Privacy Policy URL,
```

Comment:
> A URL that links to your organization's privacy policy. Privacy policies are required for apps that are Made for Kids or offer automatically renewable In-App Purchases or free subscriptions. They are also required for apps with account registration, apps that access a user’s existing account, or as otherwise required by law. Privacy policies are recommended for apps that collect user- or device-related data.